pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -154,6 +154,59 @@
     dependencies = with super; [ plenary-nvim ];
   };
 
+  avante-nvim = let
+    version = "0.0.2";
+    src = fetchFromGitHub {
+      owner = "yetone";
+      repo = "avante.nvim";
+      rev = "refs/tags/v${version}";
```

Comment:
Maybe add an update script? Looks like this plugin is in rapid development, so I doubt you will remember to update it once in a while

Reply:
Sure, done.
Well, I don't know if this will work and properly fetch the `Cargo.lock`...